src/main/src/main/java/org/geoserver/catalog/CatalogBuilder.java

```diff
@@ -1546,14 +1536,12 @@ public StyleInfo getDefaultStyle(ResourceInfo resource) throws IOException {
         }
 
         Class<?> gtype = gd.getType().getBinding();
-        if (Point.class.isAssignableFrom(gtype) || MultiPoint.class.isAssignableFrom(gtype)) {
+        if (Puntal.class.isAssignableFrom(gtype)) {
```

Comment:
LOL at first I thought this was a typo! Instead it's an interface identifying both points and multipoints.

Reply:
You both make me laugh